src/main/java/uk/gov/pay/ledger/transaction/dao/TransactionDao.java

```diff
@@ -173,18 +174,27 @@ public TransactionDao(Jdbi jdbi) {
 
     public Long getTotalForSearch(TransactionSearchParams searchParams) {
         return jdbi.withHandle(handle -> {
-            Query query = getQuery(searchParams, handle, SEARCH_COUNT_QUERY_STRING);
+            Query query = handle.configure(SqlStatements.class, stmts -> stmts.setUnusedBindingAllowed(true))
```

Comment:
Is there a reason this `setUnusedBindingAllowed` didn't need to be set before?

Reply:
there was, but I agree it's ugly and potentially dangerous, so have found a way of not needing this anymore